charts/camunda-platform-8.6/templates/camunda/_helpers.tpl

```diff
@@ -561,7 +561,11 @@ Release templates.
   {{- $baseURL := printf "%s://%s" $proto .Values.global.ingress.host }}
 
   {{- if .Values.console.enabled }}
-  {{- $baseURLInternal := printf "http://%s.%s:%v" (include "console.fullname" .) .Release.Namespace .Values.console.service.managementPort }}
```

Comment:
The changes should be in the Alpha chart.

Reply:
done